### PR TITLE
Read local settings from file

### PR DESCRIPTION
**Pull request by just-paja**
I personally hate this, but we've been stuck on this for a long time. This reads settings from local_settings.py when present. I didn't find a way on how to share variables between apache wsgi module and shell.

Currently failing because of [PR-85](https://github.com/just-paja/improtresk-api/pull/85)

### CODE REVIEW

File: improtreskApi/settings.py [hunk 1]
```diff
@@ -142,45 +139,53 @@
...
+    AUTH_LDAP_USER_SEARCH_BASE,
+    ldap.SCOPE_SUBTREE,
+    AUTH_LDAP_USER_SEARCH_FORMAT,
+)
+
+AUTH_LDAP_CACHE_GROUPS = True
+AUTH_LDAP_GROUP_CACHE_TIMEOUT = 300
+AUTH_LDAP_ALWAYS_UPDATE_USER = True
+AUTH_LDAP_FIND_GROUP_PERMS = True
+AUTH_LDAP_USER_ATTR_MAP = { "email": "mail" }
```
Comment (PetrDlouhy) [inline improtreskApi/settings.py]: remove the spaces after and before curly braces (according to Pylama)

File: improtreskApi/settings.py [hunk 2]
```diff
@@ -142,45 +139,53 @@
...
+    STATICFILES_STORAGE = 'storages.backends.s3boto.S3BotoStorage'
 
-    AUTH_LDAP_USER_FLAGS_BY_GROUP = {
-        "is_superuser": os.environ.get('DJANGO_LDAP_GROUP_SUPERUSER', ''),
-        "is_staff": os.environ.get('DJANGO_LDAP_GROUP_STAFF', ''),
-    }
+if not DEBUG:
+  X_FRAME_OPTIONS = 'DENY'
+  SECURE_BROWSER_XSS_FILTER = True
+  SESSION_COOKIE_SECURE = True
```
Comment (PetrDlouhy) [inline improtreskApi/settings.py]: make the indentation multiple of four spaces (according to Pylama)

File: improtreskApi/settings.py [hunk 3]
```diff
@@ -142,45 +139,53 @@
...
+    "(objectClass=groupOfNames)"
+)
+
+AUTH_LDAP_USER_FLAGS_BY_GROUP = {
+    "is_superuser": os.environ.get('DJANGO_LDAP_GROUP_SUPERUSER', ''),
+    "is_staff": os.environ.get('DJANGO_LDAP_GROUP_STAFF', ''),
+}
+
+try:
+    from local_settings import *
```
Comment (PetrDlouhy) [inline improtreskApi/settings.py]: add `  # noqa` to the end of this line in order to be skipped by Pylama

**Comment by PetrDlouhy** [comment]
Approved. In the book [Two scopes of Django](https://www.pdf-archive.com/2016/08/07/two-scoops-of-django-1-8/) is confirmed, that it is not possible to use environmental variables with `Apache`. They are suggesting solution to this problem in chapter `5.4` by using configuration files with the variables.
But your solution is good enough for me.

**Comment by just-paja** [comment]
@PetrDlouhy, that's good to know. Thank you.

**Pull request merged**